pom.xml

```diff
@@ -55,30 +55,29 @@
     <!-- REST AND DROPWIZARD-->
     <dropwizard.version>2.1.4</dropwizard.version>
     <dropwizard.swagger.version>2.0.12-1</dropwizard.swagger.version>
-    <zipkin.dropwizard.version>2.0.16-1</zipkin.dropwizard.version>
-    <swaggerv3.version>2.2.3</swaggerv3.version>
+    <swaggerv3.version>2.2.6</swaggerv3.version>
     <jaxb.version>2.3.6</jaxb.version>
     <jersey-media-jaxb.version>2.37</jersey-media-jaxb.version>
     <activation.version>2.1.0</activation.version>
 
     <!-- CACHE-->
-    <hazelcast.version>5.1.4</hazelcast.version>
+    <hazelcast.version>5.2.0</hazelcast.version>
```

Comment:
Shouldn't these dependencies upgrades be merged in another commit/PR ?

Reply:
Indeed!